docs/atomic.1.md

```diff
@@ -15,6 +15,10 @@ Atomic Management Tool
 # OPTIONS
 **-h** **--help**
   Print usage statement
+# ENVIRONMENT VARIABLES
+
+**ATOMIC_CONF** The location of the atomic configuration file (normally /etc/atomic.conf) can be
+overriden with the _ATOMIC_CONF_ environment variable
```

Comment:
Missing an entry for ATOMIC_CONFD here. Though I think a cleaner approach might be to just have a single ATOMIC_ETC here and construct the atomic.conf and atomic.d paths from that.